components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -97,6 +97,9 @@ def copy
       service = FinancialAssistance::Services::ApplicationService.new(application_id: params[:id])
       @application = service.copy!
       redirect_to edit_application_path(@application)
+    rescue StandardError => e
+      flash[:error] = "Failed with error - #{e}"
```

Comment:
@nisanthyaganti9
1. Add translations for the text as it displays to the user.
2. Did you mean to use **e.message** instead of **e**?